eng/testing/WasmRunnerTemplate.sh

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env bash
 
-EXECUTION_DIR=$(dirname $0)
-
-echo "Test: $1"
+JAVASCRIPT_ENGINE=$0
```

Comment:
`$0` is the name of the script so this needs to stay the way it was before and you need to make `JAVASCRIPT_ENGINE=$1` instead.

Reply:
done